test/test_distributions.py

```diff
@@ -358,6 +358,14 @@ def get_sp_dist(jax_dist):
     T(dist.Gamma, np.array([0.5, 1.3]), np.array([[1.0], [3.0]])),
     T(dist.GaussianRandomWalk, 0.1, 10),
     T(dist.GaussianRandomWalk, np.array([0.1, 0.3, 0.25]), 10),
+    T(
+        dist.GaussianCopulaBeta,
+        np.array([7.0, 2.0]),
+        jnp.array([4.0, 10.0]),
+        np.array([[1.0, 0.75], [0.75, 1.0]]),
+    ),
+    T(dist.GaussianCopulaBeta, 2.0, 1.0, np.eye(3)),
+    T(dist.GaussianCopulaBeta, 2.0, 1.0, np.full((5, 3, 3), np.eye(3))),
```

Comment:
Could you modify [this line](https://github.com/pyro-ppl/numpyro/blob/master/test/test_distributions.py#L1685) to skip checking variance of this distribution?
```
        if isinstance(d_jax, (dist.CAR, dist.GaussianCopula)):
            pytest.skip(f"{type(d_jax)} distribution does not have `variance` implemented.")
```

Reply:
pls ignore, we have `variance` now.